common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -111,8 +111,8 @@ object UrlHelpers {
 
   def getJobUrl(editionId: String): String =
     if (editionId == "au") {
-      "https://jobs.theguardian.com/landingpage/2868291/jobs-australia-html/?INTCMP=jobs_au_web_newheader"
+      "https://jobs.theguardian.com/landingpage/2868291/jobs-australia-html"
```

Comment:
AU edition does not show Search jobs in either header or footer, so this link cannot be clicked. There was a request from Guardian Australia to [remove it from the footer](https://github.com/guardian/frontend/pull/24277) (it was showing `Search UK jobs` anyway). I am not sure why this is the case for the header, it seems it was introduced [in this PR](https://github.com/guardian/frontend/pull/18987). 

For the scope of this change I am just removing the `INTCMP` link but the Australian edition will continue not showing any jobs links.